client/src/pages/open-source-skolan/[slug].js

```diff
@@ -0,0 +1,177 @@
+`
+
+const VideoIframe = styled.iframe`
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  border: 0;
+`
+
+
+
+const LessonDetails = ({ data, preview = false }) => {
+  const { data: previewData } = usePreviewSubscription(data?.lessonQuery, {
+    params: data?.queryParams ?? {},
+    initialData: data?.lesson,
+    enabled: preview,
+  })
+
+  const lesson = filterDataToSingleItem(previewData, preview)
+
+  return (
+    <>
+      {lesson && (
+        <NextSeo
+          title={lesson?.metaTags?.title ?? lesson.title}
+          titleTemplate="%s | Open Source skolan på Iteam"
+          description={
+            lesson?.metaTags?.description ?? lesson?.imageCard?.description
+          }
+          image={urlFor(
+            lesson?.metaTags?.imageWithAlt?.asset._ref ??
+              lesson?.preview?.imageCard?.image.asset._ref
+          )}
+          openGraph={{
+            title: lesson?.metaTags?.title ?? lesson.title,
+            description:
+              lesson?.metaTags?.description ?? lesson?.imageCard?.description,
+            images: [
+              {
+                url: urlFor(
+                  lesson?.metaTags?.imageWithAlt?.asset._ref ??
+                    lesson?.preview?.imageCard?.image.asset._ref
+                ),
+              },
+            ],
+            site_name: 'Iteam',
+          }}
+          twitter={{
+            title: lesson?.metaTags?.title ?? lesson.title,
+            description:
+              lesson?.metaTags?.description ?? lesson?.imageCard?.description,
+            image: urlFor(
+              lesson?.metaTags?.imageWithAlt?.asset._ref ??
+                lesson?.preview?.imageCard?.image.asset._ref
+            ),
+            handle: '@iteam1337',
+            site: '@iteam1337',
+            cardType: 'summary_large_image',
+          }}
+        />
+      )}
+      <PageWrapper footerDark>
+        {preview && <ExitPreviewLink />}
+        <Section className="pb-0">
+          <div className="pt-5"></div>
+          <Container>
+            <Row className="justify-content-center text-center">
+              <Col lg="12">
+                <Title variant="hero">{lesson?.title && lesson.title}</Title>
+                <Box className="d-flex justify-content-center">
+                  <Text>{lesson?.date && lesson.date}</Text>
+                </Box>
+              </Col>
+            </Row>
+            {lesson?.youtubeId && (
+              <Row>
+                <Col>
+                  <Box mt={96} mb={32}>
+                    <VideoContainer>
+                      <VideoIframe
+                        src={`https://www.youtube.com/embed/${lesson?.youtubeId}?autoplay=1`}
+                        title={lesson?.metaTags?.title ?? lesson.title}
+                        frameborder="0"
+                        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                        autoplay
```

Comment:
From an accessibility perspective I believe it would be better to not have autoplay.